spec/ultravisor/child/call_spec.rb

```diff
@@ -49,6 +53,7 @@ def poke_processor
 
 				# So we can check if and when it's been called
 				allow(instance).to receive(:to_s).and_call_original
+				allow(instance).to receive(:failing_method).and_call_original
```

Comment:
So much mocking .... yikes... I find these style of tests so hard to reason about. (not your fault you are just following the pattern that was there before)